### PR TITLE
test(sbb-time-input): add visual spec

### PR DESCRIPTION
**Pull request by TomMenga**

### CODE REVIEW

**Review by MarioCastigliano — APPROVED** [review]
👍

File: src/elements/time-input/time-input.visual.spec.ts [hunk 1]
```diff
@@ -0,0 +1,84 @@
+import { html, nothing, type TemplateResult } from 'lit';
```
Comment (DavideMininni-Fincons) [inline src/elements/time-input/time-input.visual.spec.ts:1]: i would add a case without prefix/suffix icons

**Review by DavideMininni-Fincons — COMMENTED** [review]
LGTM

File: src/elements/time-input/time-input.visual.spec.ts (hunk 1 above)
Comment (TomMenga) [reply src/elements/time-input/time-input.visual.spec.ts:1]: Makes sense :)

**Pull request merged**